stl/inc/format

```diff
@@ -1879,17 +1885,15 @@ struct formatter {
 
 template <class _Ty, class _CharT, _Basic_format_arg_type _ArgType>
 struct _Formatter_base {
-
     using _Pc = basic_format_parse_context<_CharT>;
 
     typename _Pc::iterator parse(_Pc& _ParseCtx) {
         _Specs_checker<_Dynamic_specs_handler<_Pc>> _Handler(_Dynamic_specs_handler<_Pc>(_Specs, _ParseCtx), _ArgType);
-        auto _It = _ParseCtx.begin();
-        _It._Seek_to(_Parse_format_specs(_ParseCtx.begin()._Unwrapped(), _ParseCtx.end()._Unwrapped(), _Handler));
-        if (_It != _ParseCtx.end() && *_It != '}') {
+        auto _It = _Parse_format_specs(_ParseCtx._Unchecked_begin(), _ParseCtx._Unchecked_end(), _Handler);
+        if (_It != _ParseCtx._Unchecked_end() && *_It != '}') {
             throw format_error("Mising '}' in format string.");
         }
-        return _It;
+        return _ParseCtx.begin() + (_It - _ParseCtx._Unchecked_begin());

```

Comment:
It would be nice to use `_RANGES _Rewrap_iterator` here, but I think it's too much work for now to move things around in other headers. Maybe future work.